### PR TITLE
Home Page messaging edits and left menu changes in Docs

### PR DESCRIPTION
**Pull request by anthonyvprakash**
**- What I did**
Home Page messaging edits and left menu changes in Docs
**- How I did it**
Added content for the new messaging

**- How to verify it**
Compare
**- Description for the changelog**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "themes/at_docsy"]
 	path = themes/at_docsy
 	url = https://github.com/atsign-foundation/at_docsy.git
+[submodule "themes/docsy"]
+	path = themes/docsy
```
Comment (Intiserahmed) [inline .gitmodules:5]: This submodule shouldn't be here..

File: resources/_gen/assets/scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68.content [hunk 2]
```diff
@@ -13640,6 +13640,17 @@ readers do not read off random characters that represent icons */
   .td-sidebar-nav > .td-sidebar-nav__section {
     padding-top: .5rem;
     padding-left: 1.5rem; }
+  .td-sidebar-nav li i {
```
Comment (Intiserahmed) [inline resources/_gen/assets/scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68.content]: Please pull the latest version both from atsign.dev and at_docsy.

**Review by Intiserahmed — CHANGES_REQUESTED** [review]
Please pull the latest version from the repos.

File: .gitmodules (hunk 1 above)
Comment (TylerTrott) [reply .gitmodules:5]: Good eye Intiser

**Pull request closed without merging**